src/main/java/io/confluent/idesidecar/restapi/models/ConnectionSpec.java

```diff
@@ -49,37 +221,271 @@ public String ccloudOrganizationId() {
 
   @Schema(description = "Configuration for Confluent Cloud connections")
   public record CCloudConfig(
-      @JsonProperty(value = "organization_id", required = true) String organizationId
+      @Schema(description = "The identifier of the CCloud organization to use. "
+                            + "The user's default organization is used when absent.")
+      @JsonProperty(value = "organization_id", required = true)
+      @NotEmpty
+      @Min(36)
+      @Max(36)
```

Comment:
Same comment as above. I think it would be great if these constraints are part of the OpenAPI spec.